app/models/tenant_subscriber.rb

```diff
@@ -0,0 +1,12 @@
+class TenantSubscriber < EventSubscriber
+
+  def do_request(event, path)
+    RestClient.post("#{tenant_url}/#{path}", event, content_type: :json)
```

Comment:
Perhaps we could convert tenant_url + do_request into a template method in superclass, so that we don't have to duplicate the rest of the logic here.